sklearn/ensemble/tests/test_stacking.py

```diff
@@ -522,3 +527,37 @@ def fit(self, X, y):
     msg = "'MyEstimator' object has no attribute 'n_features_in_'"
     with pytest.raises(AttributeError, match=msg):
         stacker.n_features_in_
+
+
+def test_clf_support_missing_values():
```

Comment:
I think that we would benefit from parametrizing for both classification and regression. Basically we don't care about asserting score or even splitting train and test. We just want a smoke test that giving X with missing values, it does not crash.

```python
@pytest.mark.parametrize(
    "stacker, X, y",
    [(StackingClassifier(...), X_clf, y_clf),
     (StackingRegressor(...), X_reg, y_reg)]
)
def test_stacking_support_missing_values(stacker, X, y):
    # introduce some missing values in X
    X = X.copy()
    X[...] = np.nan
    clf.fit(X, y).score(X, y)

```